### ISSUE TITLE
How long does it take to train the model?

### ISSUE THREAD

**Issue by jonghakim35** [mrwu-mac/EoID#4]
First of all, thanks for your work!

Can I know the approximate training time and training settings (type/number of GPUs and batch size)?

**Comment by mrwu-mac** [comment]
It takes 3 days to train the model with 4 V100 and 4 bs. After accelerating with pre-extracted CLIP logits, it takes 1.5 days.
